frontend/src/Components/Page/Header/PageHeader.js

```diff
@@ -53,7 +53,10 @@ class PageHeader extends Component {
         <div className={styles.logoContainer}>
           <Link
             className={styles.logoLink}
-            to={'/'}
+            to={{
```

Comment:
I don't think returning to Home via a link should restore scroll position, back makes sense because you're returning, but on fresh nav you're going somewhere new.